tools/manage-metadata.ts

```diff
@@ -11,6 +11,7 @@ import {
   RELEASE_NOTES_DIR,
   UNRELEASED_DIR,
 } from './constants.js'
+import isLatestReleasePreRelease from './github-preRelease-checker.js'
```

Comment:
Please revert the changes in this file. For promotions, no metadata (i.e. release notes) which are user facing are needed. Instead, we only want a notification in mattermost for ourselves. Therefore calling `notify-mattermost` with a corresponding message would be enough.
Bonus points if we include the version/a link to the github release.